### PR TITLE
Add package build-system (bs)

### PR DESCRIPTION
**Pull request by maubg-debug**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -23487,5 +23487,19 @@
     "description": "Youtube data extractor",
     "license": "MIT",
     "web": "https://github.com/thisago/ytextractor"
+  },
+  {
+    "name": "config",
```
Comment (dom96) [inline packages.json:23492]: that's a pretty generic name for a makefile-clone. Is this definitely the name you want?

Comment (maubg-debug) [reply packages.json:23492]: no, I dont know why the name is config

Comment (maubg-debug) [reply packages.json:23492]: the name I whant is `bs`

Comment (dom96) [reply packages.json:23492]: Your package name is the name of your .nimble file: https://github.com/maubg-debug/build-sys/blob/main/config.nimble

Comment (maubg-debug) [reply packages.json:23492]: aaaaaa

Comment (maubg-debug) [reply packages.json:23492]: Could u just change the name to bs and I will change the file?

Comment (maubg-debug) [reply packages.json:23492]: sry, i didntnknow that

Comment (dom96) [reply packages.json:23492]: sure:

```suggestion
    "name": "bs",
```

**Comment by dom96** [comment]
ouch, my browser now freezes when I try to rebase this. :/

**Comment by maubg-debug** [comment]
now?

**Pull request merged**